# Configuration file for the microphone
microphone:
  default_param:
    audio_format_width: 2
    chunk_size: 4800 #1600
    total_channels: 1
    audio_rate: 48000 #16000
    device_name: "CBHT Audio: USB Audio (hw:3,0)" # CBHT Audio: USB Audio (hw:2,0) , Jabra SPEAK 510 USB: Audio (hw:1,0)
    test_outdir: "$(find harmoni_microphone)/temp_data/test_example.wav"
